Update application to 4.12.4 (#374)
common-fate/terraform-aws-common-fate-deployment: .changeset/serious-years-remember.md

---
"@common-fate/terraform-aws-common-fate-deployment": patch
---

> Fixes an issue where Cedar policies for Access::Action::"Request" which restricted the resource to a specific type of entitlement would cause no entitlements to be available for request.

Reviewer: ```suggestion
Fixes an issue where Cedar policies for Access::Action::"Request" which could cause no entitlements to be available in certain circumstances.
```

Reviewer: can make this a bit less specific i think
